version.php

```diff
@@ -2,11 +2,11 @@
 
 defined('MOODLE_INTERNAL') || die();
 
-$plugin->version   = 2019111802;        // The current plugin version (Date: YYYYMMDDXX)
+$plugin->version   = 2019111807;        // The current plugin version (Date: YYYYMMDDXX)
```

Comment:
Might be a good idea to update version number to a new date (I believe the last XX bit in the YYYYMMDDXX format is just 00 for the first release that day, and increases by 1 for all subsequent releases during that day)